kpi/tests/api/v2/test_api_service_usage.py

```diff
@@ -140,81 +114,61 @@ def __add_nlp_trackers(self):
-                    'mimetype': 'video/mp4',
-                },
-                {
-                    'id': 6,
-                    'download_url': 'http://testserver/anotheruser/audio_conversion_test_image.jpg',
-                    'filename': 'anotheruser/audio_conversion_test_image.jpg',
-                    'mimetype': 'image/jpeg',
-                },
-            ],
-            '_submitted_by': 'anotheruser',
-        }
-
-        submissions.append(submission1)
-        submissions.append(submission2)
+        for x in range(count):
+            submission = {
+                '__version__': v_uid,
+                'q1': 'audio_conversion_test_clip.mp4',
+                'q2': 'audio_conversion_test_image.jpg',
+                '_uuid': str(uuid.uuid4()),
+                '_attachments': [
+                    {
+                        'id': self.attachment_id,
+                        'download_url': 'http://testserver/anotheruser/audio_conversion_test_clip.mp4',
+                        'filename': 'anotheruser/audio_conversion_test_clip.mp4',
+                        'mimetype': 'video/mp4',
+                    },
+                    {
+                        'id': self.attachment_id + 1,
+                        'download_url': 'http://testserver/anotheruser/audio_conversion_test_image.jpg',
+                        'filename': 'anotheruser/audio_conversion_test_image.jpg',
+                        'mimetype': 'image/jpeg',
+                    },
+                ],
+                '_submitted_by': 'anotheruser',
+            }
+            # increment the attachment ID for each attachment created
+            self.attachment_id = self.attachment_id + 2
+            submissions.append(submission)
 
         self.asset.deployment.mock_submissions(submissions, flush_db=False)
-        self.__update_xform_counters(self.asset, submissions=2)
+        self.update_xform_counters(self.asset, submissions=count)
 
-    def __update_xform_counters(self, asset: Asset, submissions: int = 0):
+    def update_xform_counters(self, asset: Asset, submissions: int = 0):
         """
         Create/update the daily submission counter and the shadow xform we use to query it
         """
         today = timezone.now()
         if self.xform:
             self.xform.attachment_storage_bytes += (
-                self.__expected_file_size() * submissions
+                self.expected_file_size() * submissions
             )
             self.xform.save()
         else:
             self.xform = KobocatXForm.objects.create(
                 attachment_storage_bytes=(
-                    self.__expected_file_size() * submissions
+                    self.expected_file_size() * submissions
                 ),
                 kpi_asset_uid=asset.uid,
                 date_created=today,
                 date_modified=today,
-                user_id=asset.owner_id,
+                user_id=asset.owner.id,
```

Comment:
Unintentional? It may or may not have any functional difference but why commit a change here.